app/database/group_item_store_compute_all_access.go

```diff
@@ -192,11 +178,12 @@ func (s *GroupItemStore) computeAllAccess() {
 
 	hasChanges := true
 	for hasChanges {
-		_, err = stmtInsertMissingChildren.Exec()
+		_, err = stmtDropTemporaryTable.Exec()
 		mustNotBeError(err)
-		_, err = stmtInsertMissingPropagate.Exec()
+		_, err = stmtCreateTemporaryTable.Exec()
 		mustNotBeError(err)
-		_, err = stmtUpdatePropagateAccess.Exec()
+		mustNotBeError(s.Exec(queryInsertMissingChildren).Error())
+		_, err = stmtDropTemporaryTable.Exec()
```

Comment:
Meaning your drop will be run twice in a row each time with a single create. If you want to have a "drop if exist" before your "create", just put the drop after the loop.

Reply:
fixed (did you mean 'before')?

Reply:
Moving the first one before, or the second one after, it is all the same.